lib/BC/SleighLifter.cpp

```diff
@@ -297,12 +304,17 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
   // Generic sleigh arch
   std::vector<std::string> user_op_names;
 
+  llvm::BasicBlock *entry_block;
```

Comment:
I would defensively both initialise to `nullptr`.

Reply:
It's assigned in the constructor's initialiser list already. Is this in case it gets removed from the list?

Reply:
More in case someone introduces a new ctor and forgets about this attribute.